dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/eutaxonomy/EuTaxonomyCommonFields.kt

```diff
@@ -16,6 +17,6 @@ interface EuTaxonomyCommonFields {
     val scopeOfEntities: YesNoNa?
     val nfrdMandatory: YesNo?
     val euTaxonomyActivityLevelReporting: YesNo?
-    val assurance: AssuranceData?
+    val assurance: AssuranceDataPoint<AssuranceOptions>?
```

Comment:
This could be changed if the other comment is implemented.

Reply:
Changed